lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -1050,8 +1050,8 @@ def test_key_and_dates(self, query_as):
                 ["uperf_4", "uperf_3", "uperf_2", "uperf_1", "test", "fio_2", "fio_1"],
             ),
             (
-                # Sort by date timestamp
-                "dataset.uploaded",
+                # Sort by date timestamp (redundantly typing as date)
+                "dataset.uploaded:asc:date",
```

Comment:
I'm not sure I understand the amendment to the comment:  is it "redundant" because the typing is going to force a conversion from "date" to "date"?

Wouldn't it be better to say something like "explicitly" rather than "redundantly"?

Reply:
Yes to the first. The native SQL column `Dataset.uploaded` is a `DATETIME` already, so we're redundantly casting a datetime to a datetime. (And we *do* have enough column metadata that I *think* I could have suppressed a redundant cast, but it didn't seem worth the effort.)

So it's both explicit and, more importantly, redundant.